OpenRA.Mods.Common/Traits/Render/WithRepairOverlay.cs

```diff
@@ -18,9 +18,15 @@ namespace OpenRA.Mods.Common.Traits.Render
 	[Desc("Displays an overlay when the building is being repaired by the player.")]
 	public class WithRepairOverlayInfo : PausableConditionalTraitInfo, Requires<RenderSpritesInfo>, Requires<BodyOrientationInfo>
 	{
-		[Desc("Sequence name to use")]
+		[Desc("Sequence to use upon repair beginning.")]
+		[SequenceReference("Image")] public readonly string StartSequence = null;
+
+		[Desc("Looping Sequence name to use.")]
```

Comment:
This is only true if `StartSequence` is non-null.  If it is null then it is played once (not looping) for each repair tick.  This isn't very nice.